FirebaseVertexAI/Sources/Types/Public/Schema.swift

```diff
@@ -20,19 +20,38 @@ import Foundation
 /// [OpenAPI 3.0 schema object](https://spec.openapis.org/oas/v3.0.3#schema).
 public class Schema {
   /// Modifiers describing the expected format of a string `Schema`.
-  public enum StringFormat {
+  public struct StringFormat: EncodableProtoEnum {
+    enum Kind: String {
+      case custom // TODO: Remove when we have at least one specific string format.
+    }
+
     /// A custom string format.
-    case custom(String)
+    public static func custom(_ format: String) -> StringFormat {
+      return self.init(rawValue: format)
+    }
+
+    let rawValue: String
   }
 
   /// Modifiers describing the expected format of an integer `Schema`.
-  public enum IntegerFormat {
+  public struct IntegerFormat: EncodableProtoEnum {
+    enum Kind: String {
+      case int32
+      case int64
+    }
+
     /// A 32-bit signed integer.
-    case int32
+    public static let int32 = IntegerFormat(kind: .int32)
```

Comment:
Somehow I just realized now that I could be using `public static let` instead of computed properties. Will update all the rest at once in a follow-up PR.